query-engine/connector-test-kit/src/test/scala/writes/ids/relation_pks/CompoundPKRelationFieldSpec.scala

```diff
@@ -14,252 +14,6 @@ import util._
 //
 // 2) Checks basic cursor functionality.
 class CompoundPKRelationFieldSpec extends FlatSpec with Matchers with ApiSpecBase {
-  // Mutations in this test:
-  //  create         | root   | checked
-  //  update         | root   | checked
-  //  delete         | root   | not possible (1!:1!)
-  //  upsert         | root   | checked
-  //  updateMany     | root   | unnecessary
-  //  deleteMany     | root   | unnecessary
-  //  nested create  | create | checked
-  //  nested update  | update | checked
-  //  nested connect | -      | not possible (1!:1!)
-  //  nested disconn | -      | not possible (1!:1!)
-  //  nested delete  | -      | not possible (1!:1!)
-  //  nested set     | -      | not possible (1!:1!)
-  //  nested upsert  | -      | not possible (1!:1!)
-  //  nested deleteM | -      | not possible (1!:1!)
-  //  nested updateM | -      | not possible (1!:1!)
-  "Using a compound ID that includes a 1!:1! single-field relation" should "work" in {
```

Comment:
Should we bring this one back? The essence of the test is that `@@id` contains a relation scalar. The fact that it is a required one to one does not seem important.